BuildTransportPackage.ps1

```diff
@@ -0,0 +1,170 @@
+Param(
+    [string]$Platform,
+    [string]$Configuration,
+    [string]$LocalPackagesPath = $null,
+    [string]$UpdateVersionDetailsPath = $null
+)
+
```

Comment:
Help command would be useful here.

We also need to warn developers that what nuget.config points to depends on whether the branch is develop or main

Reply:
Why would developers need to be warned about the different nuget.config files?  Are they different?

Reply:
Nuget.config in develop points to the Internal feed whereas main points to an external feed. 
Someone building Foundations should be aware.

Reply:
Done.